dev/com.ibm.ws.webcontainer/src/com/ibm/wsspi/webcontainer/WCCustomProperties.java

```diff
@@ -312,6 +312,9 @@
 
     //19.0.0.8
     public static boolean GET_REAL_PATH_RETURNS_QUALIFIED_PATH;
+    
+    //20.0.0.6
```

Comment:
Just as a note we'll want to update this as this won't make 20006

Reply:
done